internal/exec/resolvable/resolvable.go

```diff
@@ -41,15 +41,20 @@ type Object struct {
 
 type Field struct {
 	ast.FieldDefinition
-	TypeName          string
-	MethodIndex       int
-	FieldIndex        []int
-	HasContext        bool
-	HasError          bool
-	ArgsPacker        *packer.StructPacker
-	DirectiveVisitors []directives.ResolverInterceptor
-	ValueExec         Resolvable
-	TraceLabel        string
+	TypeName    string
+	MethodIndex int
+	FieldIndex  []int
+	HasContext  bool
+	HasError    bool
+	ArgsPacker  *packer.StructPacker
+	Visitors    *FieldVisitors
+	ValueExec   Resolvable
+	TraceLabel  string
+}
+
+type FieldVisitors struct {
+	Interceptors []directives.ResolverInterceptor
```

Comment:
I wonder if we should call these `ResolverMiddleware` or `ResolverInterceptor` 🤔